src/main/java/practice/StreamPractice.java

```diff
@@ -13,17 +16,33 @@ public class StreamPractice {
      * If there is no needed data throw RuntimeException with message
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
+
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(line -> Arrays.stream(line.split(",")))
+                .map(Integer::parseInt)
+                .filter(n -> n % 2 == 0)
+                .min(Integer::compare)
```

Comment:
```suggestion
                .min()
```
It’s  default behavior